scripts/verify-replace.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+TMP_DIR=/tmp/
+BASE_REPO_PATH=$(mktemp -d ${TMP_DIR}replace-verify.XXX)
+GH_BASE_URL_KS=https://github.com/kubesaw/
+GH_BASE_URL_CRT=https://github.com/codeready-toolchain/
+declare -a repos=("${GH_BASE_URL_KS}ksctl" "${GH_BASE_URL_CRT}host-operator" "${GH_BASE_URL_CRT}member-operator" "${GH_BASE_URL_CRT}registration-service" "${GH_BASE_URL_CRT}toolchain-e2e")
+C_PATH=${PWD}
+ERRORLIST=()
+
+echo Initiating verify-replace on dependent repos
+for repo in "${repos[@]}"
+do
+    echo =========================================================================================
+    echo  
+    echo                        "$(basename $repo)"
+    echo                                                                     
+    echo =========================================================================================                                            
+    REPO_PATH=$BASE_REPO_PATH/$(basename $repo)
```

Comment:
the best practice is to always use curly brackets for variables in bash to avoid unexpected wrong expansion of variables
```suggestion
    REPO_PATH=${BASE_REPO_PATH}/$(basename ${repo})
```
could you please update it in the rest of the script?

Reply:
done